README.md

```diff
@@ -29,20 +29,28 @@ See the [中文文档](https://opensergo.io/zh-cn/) for document in Chinese.
 
 2. Subscribe data from OpenSergo control plane:
 
-```java
+``` java
 // 1. Create client with remote host and port of OpenSergo control plane
 OpenSergoClient client = new OpenSergoClient(host, port);
 // 2. Start the OpenSergo client.
 client.start();
 
-// 3. Subscribe the config of the target (namespace, appName, kind)
-client.subscribeConfig(new SubscribeKey("default", "my-service", configKind),
-    new OpenSergoConfigSubscriber() {
+// 3. Prepare for subscribing the config of the target (namespace, appName, kind)
+// 3.1 Create the SubscribeKey of your target
+SubscribeKey subscribeKey = new SubscribeKey("default", "my-service", configKind);
+// 3.2 Create the SubscribeInfo of your target by subscribeKey
+OpensergoClientSubscribeInfo subscribeInfo = new OpensergoClientSubscribeInfo(subscribeKey);
```

Comment:
Could you please illustrate why we need a `OpensergoClientSubscribeInfo` here?

Reply:
`OpensergoClientSubscribeInfo` contains `SubscribeKey` and `List<OpenSergoConfigSubscriber>`, means one SubscribeKey can add several OpenSergoConfigSubscriber, which will help user to subscribeConfig with several  OpenSergoConfigSubscriber at one time. 

OpensergoClientSubscribeInfo will used in the following func `registerSubscribeInfo(OpensergoClientSubscribeInfo)`, it also can be separated to `SubscribeKey` and `List<OpenSergoConfigSubscriber>` in registerSubscribeInfo() if necessary.

Reply:
`OpensergoClientSubscribeInfo.java` has been removed